README.md

```diff
@@ -26,16 +26,17 @@ libp2p-switch is used by [libp2p](https://github.com/libp2p/js-libp2p) but it ca
 - [Usage](#usage)
   - [Create a libp2p switch](#create-a-libp2p-switch)
 - [API](#api)
-  - [`switch.dial(peer, protocol, callback)`](#swarmdialpi-protocol-callback)
-  - [`switch.hangUp(peer, callback)`](#swarmhanguppi-callback)
-  - [`switch.handle(protocol, handler)`](#swarmhandleprotocol-handler)
-  - [`switch.unhandle(protocol)`](#swarmunhandleprotocol)
-  - [`switch.start(callback)`](#swarmlistencallback)
-  - [`switch.stop(callback)`](#swarmclosecallback)
-  - [`switch.connection`](#connection)
+  - [`switch.dial(peer, protocol, callback)`](#switchdialpeer-protocol-callback)
+  - [`switch.dialFSM(peer, protocol, callback)`](#switchdialfsmpeer-protocol-callback)
+  - [`switch.hangUp(peer, callback)`](#switchhanguppeer-callback)
+  - [`switch.handle(protocol, handler)`](#switchhandleprotocol-handler)
+  - [`switch.unhandle(protocol)`](#switchunhandleprotocol)
+  - [`switch.start(callback)`](#switchstartcallback)
+  - [`switch.stop(callback)`](#switchstopcallback)
+  - [`switch.connection`](#switchconnection)
```

Comment:
Can we please alphabetize the API? I believe this helps when looking for documentation.